composer.json

```diff
@@ -8,19 +8,19 @@
     "ext-json": "*",
     "pimcore/pimcore": "^11.0",
     "psr/log": "^3.0",
-    "ruflin/elastica": "8.0.0-alpha1",
-    "symfony/console": "^6.2",
-    "symfony/lock": "^6.2"
+    "ruflin/elastica": "^8.0.0",
```

Comment:
As the [changes between the last alpha and the stable tag](https://github.com/ruflin/Elastica/compare/8.0.0-alpha1...8.0.0) are minimal and our `4.x` versions always [used the then-current v8 of `ruflin/elastica`](https://github.com/valantic/pimcore-elastica-bridge/blob/v4.0.0-beta.1/composer.json#L11), I'd argue we can release this as a minor version.